modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/config/Config.scala

```diff
@@ -119,8 +119,11 @@ object Config {
     loading: FiniteDuration,
     nonLoading: FiniteDuration,
     sqsVisibility: FiniteDuration,
-    rollbackCommit: FiniteDuration
-  )
+    rollbackCommit: FiniteDuration,
+    connectionIsValid: FiniteDuration
+  ) {
+    def totalTimeToRollBack: FiniteDuration = rollbackCommit + connectionIsValid
```

Comment:
I'm not completely happy with this line, because it's mixing logic between two distant parts of the code (StateMonitoring and doobit commit strategy).  However... if this fix works... if there are no more "loader is stuck" errors, then we can follow-up by removing the StateMonitoring feature in the next release.  It will remove a lot of complexity.